opencxl/cxl/transport/transaction.py

```diff
@@ -984,6 +984,9 @@ class CxlCacheD2HReqPacket(CxlCacheBasePacket):
     def get_address(self) -> int:
         return self.d2hreq_header.addr << 6
 
+    def set_cache_id(self, cache_id: int):
+        self.d2hreq_header.cache_id = cache_id
+
```

Comment:
Unlike `get_address()`, this is just a simple assignment of the content to the packet field. No need to make it into a function. Remove.